### ISSUE TITLE
No detection when I train the Model on balloon dataset, but with provided balloon h5 data, everything's fine

### ISSUE THREAD

**Issue by HeGhK** [matterport/Mask_RCNN#969]
Hello everyone,

Thanks to author of this fine code/model. I am facing a problem when I try to train the model with provided balloon dataset. The training is within 30 min. through, but when I follow the provided notebook, namely "inspect_balloon_model.ipynb", I get zero detection. But when I specify the provided balloon weights h5 data as model, everything works fine and I can reproduce the model inspection notebook without a single problem.  I would appreciate any feedback regarding my problem. Thanks in advance.
PS: I should mention that I use docker and latest docker image of tensorflow.
IDE: pycharm
Environment: official docker image of tensorflow (tensorflow:latest-gpu-py3)

**Comment by k30r88** [comment]
hi @HeGhK i have similar issues, but even without weights h5 , our model could not detect any balloon. Can you provide any recommendation ?

**Comment by SamGhK** [comment]
@k30r88 Hi, unfortunately, I couldn't solve my problem regarding the balloon data. But for my own data, I could use the model without a single problem. I used the model for airbus ship detection kaggle challenge. I wrote my own data loader and everything ran smoothly. My guess was, that the problem lied in provided data loader code, somewhere, which I did not have the time to debug. If it helps, I could provide you with my own data loader for ship detection.
